WordPress/src/main/java/org/wordpress/android/ui/main/MeFragment.kt

```diff
@@ -151,7 +158,12 @@ class MeFragment : Fragment(R.layout.me_fragment), OnScrollToTopListener {
             }
         }
 
-        viewModel = ViewModelProvider(this@MeFragment, viewModelFactory).get(MeViewModel::class.java)
+        if (viewModel.unifiedAboutFeatureConfig.isEnabled()) initUnifiedAboutUiState() else initRecommendUiState()
```

Comment:
I wonder if it wouldn't be less confusing to just have a separate row for the about item in the layout instead of reusing the "recommend" one. Then we could move this logic back into `setupViews` and have something like:

```kotlin
if (unifiedAboutFeatureConfig.isEnabled()) {
    rowAbout.setOnClickListener {
        viewModel.showUnifiedAbout()
    }
} else {
    initRecommendUiState()
}
```

It seems simpler for me, but let me know what you think!

Reply:
Both require an if else block.  I'm not sure it makes any difference.  Eventually, we need to remove share option from here anyways

Reply:
>I'm not sure it makes any difference.

It's _very likely_ that it won't make a difference, but I do see a difference in terms of both readability and maintainability:
- I'd argue that since the code above is similar to what we are already doing for other items, it's probably easier for someone to understand what's going on. We would also avoid unnecessary code since we wouldn't need to call `setRecommendLoadingState(false)` as the About item doesn't need to be wrapped with the shimmer layout. 
- I also understand this is all obvious for us right now, but we're dealing with a huge codebase here, so we should always consider we're not the only ones maintaining this code (and even so, it's always possible we're not the ones maintaining this code in the future), and it may not be as obvious then.